department-of-veterans-affairs/vets-api
# [67016] Enable terms of use for va.gov staging and below

## Summary

- This PR enables terms of use for va.gov staging and below, when `oauth=false` is set in the sign up modal

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#67016
- *Link to ticket created in va.gov-team repo*
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*


## Testing done

- Authenticate with `oauth=false`
- Confirm that Terms of Use page shows up
- Accept terms, log out, reauthenticate
- Confirm that terms of use page does not show up

## What areas of the site does it impact?
Authentication

## Acceptance criteria

- [ ] Confirm that Terms of Use Page shows up during `oauth=false` authentications
